### PR TITLE
generating command line docs automatically

### LINKED ISSUE #722: sealer need to support generating command line docs automatically
## Issue Description

Type: *feature request*

### Describe what feature you want

sealer need to support generating command line docs automatically.

### PR DESCRIPTION
**Pull request by TheBeatles1994**
### Describe what this PR does / why we need it

generating command line docs automatically.

### Does this pull request fix one issue?

Fixes #722 

### Describe how you did it

Cobra supports generating documentation for commands.

### Describe how to verify it

```bash
cd $GOPATH/src/github.com/alibaba/sealer
make build
_output/bin/sealer/darwin_amd64/sealer gen-doc --path docs/commandline
```

### CODE REVIEW

File: sealer/cmd/images.go [hunk 1]
```diff
@@ -60,6 +60,7 @@ var listCmd = &cobra.Command{
 }
 
 func init() {
+	listCmd.DisableAutoGenTag = true
```
Comment (allencloud) [inline sealer/cmd/images.go:63]: We should apply the rule of DRY(Don't Repeat yourself). Actually doing command execution in every command's init() function is not a good way.

**Comment by allencloud** [comment]
We encourage so much the principle of `code is doc`. 👍  🦁  @TheBeatles1994

File: sealer/cmd/images.go (hunk 1 above)
Comment (TheBeatles1994) [reply sealer/cmd/images.go]: Done. I see that only one command `rootCmd.DisableAutoGenTag=true` is enough. 

Dry is so important that avoid repetition of codes!

**Comment by fanux** [comment]
What do mean about `maintaining these to a much higher level?` ?

**Pull request merged**